### ISSUE TITLE
The host cannot be found.

### ISSUE THREAD

**Issue by Froussios** [chromium/eclipse-gn#1]
The host http://gneditor.murzwin.com/snapshot does not appear to exist anymore.

**Comment by chaijunkun** [comment]
Yes, I met the same issue. Is there anybody know another avaliable site for installing this plugin?

**Comment by chaijunkun** [comment]
I finally got it! Actually, this is a maven project. you can use
git clone https://github.com/chromium/eclipse-gn.git
to download source code. then use
mvn package
to compile and package.
As described in README,, you can find the result located in target directory of  org.chromium.gnsupport.site module.
for example, my clone location is: D:\project\eclipse-gn,
open your eclipse, Help -> Install New Software, click 'Add' button to add a repository.
you can use 'gn' as the repository, it doesn't mater.
but, the location you should fill as 'file:/D:/project/eclipse-gn/org.chromium.gnsupport.site/target/repository/'
click 'OK' button, congratulation, you can see 'GN Meta-Build System Support'
click Next -> Next.... as normal plugin installing progress.
